src/pr-info.ts

```diff
@@ -171,7 +164,11 @@ export async function queryPRInfo(prNumber: number) {
 }
 
 // The GQL response -> Useful data for us
-export async function deriveStateForPR(info: ApolloQueryResult<PRQueryResult>): Promise<PrInfo | BotFail | BotNOOP>  {
+export async function deriveStateForPR(
+    info: ApolloQueryResult<PRQueryResult>,
+    getOwners?: (packages: readonly string[]) => OwnerInfo | Promise<OwnerInfo>,
+    getDownloads?: (packages: readonly string[]) => Record<string, number> | Promise<Record<string, number>>
```

Comment:
Getting owners and downloads uses simple DI instead of jest mocks so it can be leveraged when updating fixtures as well.

Reply:
big win